ext/util/types.c

```diff
@@ -0,0 +1,657 @@
+#include "php_cassandra.h"
+#include "util/types.h"
+#include <ext/standard/php_smart_str.h>
+#include "../src/Cassandra/Bigint.h"
```

Comment:
You shouldn't need the relative parent paths here; root include search path should be the *ext* directory. In addition it would be good to add *src/Cassandra* to the include path in the config.m4 (already exists for Windows config.w32). With that the paths could be removed from the include statement.

Since header guards are used for all the Cassandra/*.h there will not be issues with system headers (e.g. float.h, math.h, ...etc). I would even suggest just include the *src* directory in the include search path and then the include line could be `#include "Cassandra/Float.h"`; as to not confuse anybody reading through the code and mistaken `Float.h` for `float.h` (which is more of an issues for case-insensitive operating systems; WIndows for example)